code/modules/projectiles/ammunition/energy/ebow.dm

```diff
@@ -1,7 +1,7 @@
 /obj/item/ammo_casing/energy/bolt
 	projectile_type = /obj/projectile/energy/bolt
 	select_name = "bolt"
-	e_cost = 500
+	e_cost = LASER_SHOTS(1, 500)
```

Comment:
```suggestion
	e_cost = LASER_SHOTS(1, STANDARD_CELL_CHARGE * 0.5)
```
The macro's been made to explicify the amount of shots available, so my guess the second arg should be this or a new define for kinetic accelerator cells (i think that's what e-bows use), to avoid confusion.